ansible_community_steering_committee.rst

```diff
@@ -79,8 +79,12 @@ The quorum, the minimum number of Committee members who must vote on a topic in
 
 If the quorum number contains a fractional part, it is rounded up to the next whole number. For example, if there are thirteen members currently in the committee, the quorum will be seven.
 
+Votes must always have "no change" as an option.
+
 In case of equal numbers of votes for and against a topic, the chairperson's vote will be used as a casting vote to resolve the deadlock. For example, if there are six votes for and six votes against a topic, and the chairperson's vote is among those six which are for the topic, the final decision will be positive. If the chairperson has not voted yet, other members ask them to vote.
 
+For votes with more than two options, one choice must have at least half of the votes. If two choices happen to both have half of the votes, the chairperson's vote will be used as a casting vote. If no choice has at least half of the votes, the vote choices have to be adjusted so that a majority can be found for a choice in a new vote.
```

Comment:
> If no choice has at least half of the votes, the vote choices have to be adjusted so that a majority can be found for a choice in a new vote.

I'm fine with leaving this as is because it's probably rare, but it is somewhat ambiguous as to _how_ that adjustment happens.

We could do a runoff type of thing, and just eliminate the choice with fewest votes (but instead of automatically reallocating we have a new vote like already suggested).

Anyway this is totally non-blocking, it can even be decided per vote how to handle the situation, since again I think it's probably going to be rare.

Reply:
I think it is not necessary to specify how to exactly proceed, since it will depend a lot on the situation (what the choices are). Maybe a new compromise can be found and voted on. In any case, the new selection is voted on again, and keeping status quo is still an option in it, so if a popular choice was kicked off, people will vote against it. So if you actually want the vote to eventually pass, you better leave away options that weren't too popular, or try to find a compromise.

Reply:
OK, we'll leave it as is or someone has other suggestions?:) I'm personally fine with this as is.